components/concept/engine/src/main/java/mozilla/components/concept/engine/content/blocking/Tracker.kt

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.content.blocking
+/**
+ * Represents a blocked content.
```

Comment:
nit: a blocked content tracker? Let's also document the params.

Reply:
Fixed!